packages/blaze/events.js

```diff
@@ -178,29 +194,28 @@ EventSupport.listen = function (element, events, selector, handler, recipient, g
 
   return {
     // closes over just `element` and `newHandlerRecs`
-    stop: function () {
-      var eventDict = element.$blaze_events;
-      if (! eventDict)
-        return;
+    stop() {
+      const eventDict = element.$blaze_events;
+      if (!eventDict) return;
       // newHandlerRecs has only one item unless you specify multiple
       // event types.  If this code is slow, it's because we have to
       // iterate over handlerList here.  Clearing a whole handlerList
       // via stop() methods is O(N^2) in the number of handlers on
       // an element.
-      for (var i = 0; i < newHandlerRecs.length; i++) {
-        var handlerToRemove = newHandlerRecs[i];
-        var info = eventDict[handlerToRemove.type];
-        if (! info)
-          continue;
-        var handlerList = info.handlers;
-        for (var j = handlerList.length - 1; j >= 0; j--) {
+      for (let i = 0; i < newHandlerRecs.length; i++) {
```

Comment:
We should try to avoid `continue` where possible. A potential alternative is to use `newHandlerRecs.forEach` and use `if (!info) return;` because `return` actually only ends the function early and pops it from the call stack, `continue` however creates `goto` statements